exec/java-exec/src/test/java/org/apache/drill/exec/store/json/TestJsonRecordReader.java

```diff
@@ -179,4 +181,28 @@ public void testNestedFilter() throws Exception {
         .sqlBaselineQuery(baselineQuery)
         .go();
   }
+
+
+ @Test // See DRILL-4653
+  public void testSkippingInvalidJSONRecords() throws Exception {
```

Comment:
For both these tests could you pls use the testBuilder() framework ?  This is the recommended way to write the unit tests .. you can see one of the other tests in this file.